dts/arm/unisoc/mem.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018, UNISOC Incorporated
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __DT_UNISOC_MEM_H
+#define __DT_UNISOC_MEM_H
+
+#define __SIZE_K(x) (x * 1024)
```

Comment:
We've moved away from this style of mem.h header.  We introduced DT_SIZE_K/DT_SIZE_M.

Reply:
done.